continuous_integration/recipe/meta.yaml

```diff
@@ -42,7 +42,9 @@ test:
 about:
   home: https://github.com/dask/dask/
   license: BSD-3-Clause
-  license_file: LICENSE.txt
+  license_file:
+    - LICENSE.txt
+    - dask/array/NUMPY_LICENSE.txt
```

Comment:
We should add this to conda-forge. Raised issue ( https://github.com/conda-forge/dask-core-feedstock/issues/128 ) to address. Can handle as part of the Dask release (as it will depend on this PR)

Reply:
cc @charlesbluca (for awareness)